app/models/item.rb

```diff
@@ -6,4 +6,6 @@ class Item < ActiveRecord::Base
   validates :price, numericality: { greater_than: 0 }
   has_many :order_items
   has_many :orders, through: :order_items
+  has_attached_file :image, :default_url => "/images/default.jpg"
+  validates_attachment_content_type :image, :content_type => [ "image/jpg", "image/jpeg", "image/png" ]
```

Comment:
Line is too long. [103/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space inside square brackets detected.